Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml

```diff
@@ -274,8 +274,30 @@
 - type: entity
   parent: EncryptionKey
   id: EncryptionKeySyndie
-  name: blood-red encryption key
-  description: An encryption key used by... wait... Who is owner of this chip?
+  name: syndicate encryption key
+  description: An encryption key used by Syndicate agents to not only access the station's radio channels, but also securely contact other agents in the field.
```

Comment:
Very bad desc and name. Description of abilities of it should be in uplink. This is just appearance of item, not tutorial how to ~~troll security~~ use this item